src/index.rs

```diff
@@ -405,9 +405,9 @@ impl Index {
   ///   writeTree(): void;
   /// }
   /// ```
-  pub fn write_tree(&mut self) -> crate::Result<String> {
-    let id = self.inner.write_tree().map(|x| x.to_string())?;
-    Ok(id)
```

Comment:
I think we should follow the libgit2 return type. Instead of returning `void`, how about changing the documentation to return a `string` type?

Reply:
Yes. I'll do that.